### ISSUE TITLE
[GUI] QT App Build process

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#33]
## User story
1. As a developer
2. I want to automate the build process for the Qt project
3. So that we can efficiently manage and maintain the build process and its docmentation

## Acceptance criteria
* .exe file created
* Document prerequisites, build process
* Automate as much as is useful

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
